src/shared/ui/AnswersItem/AnswersItem.jsx

```diff
@@ -1,17 +1,41 @@
+import React from 'react';
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Typography } from 'shared/ui';
 import styles from './AnswersItem.module.scss';
 
 function AnswersItem({ data, mode }) {
+	const [isOpen, setIsOpen] = React.useState(false);
+
+	const contentClassName = classNames(styles[`content-${mode}`], {
+		[styles.open]: isOpen,
+	});
+	const arrowClassName = classNames(styles[`arrow-${mode}`], {
+		[styles.rotate]: isOpen,
+	});
+
 	return (
-		<li className={styles[mode]}>
-			<h3 className={styles[`title-${mode}`]}>{data.title}</h3>
-			<p className={styles[`content-${mode}`]}>{data.content}</p>
-		</li>
+		<div className={styles[mode]}>
+			<div className={styles[`wrapper-${mode}`]}>
+				<Typography
+					variant="body1"
+					style={{ cursor: 'pointer' }}
+					onClick={() => setIsOpen(!isOpen)}
```

Comment:
При смене смене state всегда лучше использовать prev т.к. там всегда актуальные данные 

```suggestion
					onClick={() => setIsOpen( prev => !prev)}
```